db/schema.rb

```diff
@@ -358,6 +358,8 @@
     t.datetime "updated_at", null: false
     t.bigint "document_id"
     t.bigint "whitehall_migration_id"
+    t.text "integrity_check_problems"
```

Comment:
Can we do this as a postgres array of strings? https://guides.rubyonrails.org/active_record_postgresql.html#array

Reply:
I was following the pattern used on the users table, where a text field is used for `permissions` and then it's serialised in the model, but I'm happy to change it.